dish_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './dish'
+require 'pry'
+
+class DishTest < Minitest::Test
```

Comment:
Tests look great!  Do try to test more than one Dish so that you know that other categories will also work!